pilot/pkg/xds/ecds.go

```diff
@@ -59,6 +59,14 @@ func ecdsNeedsPush(req *model.PushRequest) bool {
 	return false
 }
 
+// Check if the secret updates is relevant to Wasm image pull. If not relevant, skip pushing ECDS.
+// this occurs when user only update the Secret referenced by WasmPlugin/EnvoyFilter.
+func secretConfigsUpdatedOnly(req *model.PushRequest) bool {
```

Comment:
I would prefer also to abstract `updatedSecrets` cal into this function and rename it, as it is very confusing

Reply:
updated, PTAL.